packages/eslint-plugin-jest/src/rules/types.js

```diff
@@ -8,31 +8,37 @@
  * @flow
  */
 
-export type Identifier = {|
+type Node = MemberExpression | CallExpression;
```

Comment:
I started doing this, but my flow-fu failed me. Didn't know it'd be as little as this!